tests/test_lib.py

```diff
@@ -23,5 +23,8 @@ def setUpClass(self):
     def tearDownClass(self):
         self.csvfile.close()
 
-    def test_csv_fieldnames(self):
-      assert 1 == 2
+    def setup_method(self, method):
+        self.reader = csv.DictReader(open(self.csvfile.name))
+
+    def test_csv_field_names(self):
```

Comment:
I think in light of my comments on the previous commit, you'll probably want to rewrite the tests introduced in this commit and any later ones. I hope that's not too much hassle - let me know if I can help with that.